packages/contracts/src/MultisigSetup.sol

```diff
@@ -105,23 +170,47 @@ contract MultisigSetup is PluginUpgradeableSetup {
         SetupPayload calldata _payload
     ) external view returns (PermissionLib.MultiTargetPermission[] memory permissions) {
         // Prepare permissions
-        permissions = new PermissionLib.MultiTargetPermission[](2);
+        permissions = new PermissionLib.MultiTargetPermission[](5);
 
         // Set permissions to be Revoked.
         permissions[0] = PermissionLib.MultiTargetPermission({
             operation: PermissionLib.Operation.Revoke,
             where: _payload.plugin,
             who: _dao,
             condition: PermissionLib.NO_CONDITION,
-            permissionId: Multisig(IMPLEMENTATION).UPDATE_MULTISIG_SETTINGS_PERMISSION_ID()
+            permissionId: UPDATE_MULTISIG_SETTINGS_PERMISSION_ID
         });
 
-        permissions[1] = PermissionLib.MultiTargetPermission({
+        permissions[1] = PermissionLib.MultiTargetPermission(
+            PermissionLib.Operation.Revoke,
+            _payload.plugin,
+            _dao,
+            PermissionLib.NO_CONDITION,
+            UPGRADE_PLUGIN_PERMISSION_ID
+        );
```

Comment:
from `v1.3` the `UPGRADE_PLUGIN_PERMISSION_ID` is not being granted to the DAO, so it doesn’t need to be revoked when uninstalling. 
It is already being [revoked when upgrading from a previous version](https://github.com/aragon/multisig-plugin/blob/6a2c080836fa3340d8dc0b37c78f9225a1c2ee7b/packages/contracts/src/MultisigSetup.sol#L134-L140)